ScnSoft-Education/LeetCodeTraining.Tests/PalindromeNumberSolutionTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using LeetCodeTraining.PalindromeNumber;
+
+namespace LeetCodeTraining.Tests
+{
+    public class PalindromeNumberSolutionTests
+    {
+        [Theory]
+        [InlineData(-123, false)]
+        [InlineData(0, true)]
+        [InlineData(1, true)]
+        [InlineData(121, true)]
```

Comment:
Please add tests  for number  -121. and 10 as on TestCase tab.

![image](https://github.com/daniilnaumovich/ScnSoft-Education/assets/101803780/aff54245-fc03-4c0d-b883-da714fd2f5c9)

Jeneral note:
As soon as changes from code review suggestions are applied within an PR with  please then always send the request to recheck the PR.